.github/workflows/pypi-workflow.yml

```diff
@@ -0,0 +1,47 @@
+name: build
+
+on:
+  push:
+    branches:
+    - master
+
+jobs:
+  publish:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout 🛎️
+        uses: actions/checkout@v2
+      - name: Set up Python ${{ matrix.python-version }} 🔧
+        uses: actions/setup-python@v2
+        with:
+          python-version: 3.9
+      - name: Install dependencies ⚙️
+        run: |
+          sudo apt-get install qpdf
+          python -m pip install --upgrade pip
+      - name: Statically checking code 🔎
+        run: |
+          pip install . pylint
+          pylint fpdf
+      - name: Ensure code has been autoformatted with black 🖌️
+        run: |
+          pip install black
+          black --check .
+      - name: Running tests ☑
+        run: |
+          pip install pytest
+          pytest
+      - name: Publish 🚀
+        run: |
+          apt install curl jq -y || apt update && apt install curl jq -y
```

Comment:
Why not always performing the `apt update` ?

Reply:
in case apt is initialized in their image, was planning on experimenting and finding out